apps/condo/domains/settings/components/ticketSubmitting/OnlyProgressionMeterReadingsForm.tsx

```diff
@@ -0,0 +1,126 @@
+import { MobileFeatureConfig as MobileFeatureConfigType } from '@app/condo/schema'
+import { Col, Form, Row, Typography } from 'antd'
+import { Gutter } from 'antd/es/grid/row'
+import get from 'lodash/get'
+import { useRouter } from 'next/router'
+import React, { useEffect, useMemo, useState } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { ActionBar, Button, Checkbox } from '@open-condo/ui'
+
+import { FormWithAction } from '@condo/domains/common/components/containers/FormList'
+import { MobileFeatureConfig } from '@condo/domains/settings/utils/clientSchema'
+
+const INPUT_LAYOUT_PROPS = {
+    labelCol: {
+        span: 24,
+        md: 5,
+    },
+    wrapperCol: {
+        span: 24,
+        md: 6,
+    },
+    styled: {
+        paddingBottom: '12px',
+    },
+    colon: false,
+}
+const BIG_ROW_GUTTERS: [Gutter, Gutter] = [0, 60]
+const MIDDLE_ROW_GUTTERS: [Gutter, Gutter] = [0, 40]
+const SMALL_ROW_GUTTERS: [Gutter, Gutter] = [0, 20]
+
+interface ITicketSubmittingSettingsForm {
+    mobileConfig?: MobileFeatureConfigType,
+    userOrganizationId: string,
+}
+
+export const OnlyProgressionMeterReadingsForm: React.FC<ITicketSubmittingSettingsForm> = ({ mobileConfig, userOrganizationId }) => {
+    const intl = useIntl()
+    const SaveMessage = intl.formatMessage({ id: 'Save' })
+    const MessageAboutFeat = intl.formatMessage({ id: 'pages.condo.settings.mobileFeatureConfig.OnlyProgressionMeterReadings.messageAboutFeat' })
+    const EnableMessage = intl.formatMessage({ id: 'pages.condo.settings.mobileFeatureConfig.OnlyProgressionMeterReadings.isEnabled' })
+
+    const router = useRouter()
+    const [onlyProgressionMeterReadingsIsEnabled, setOnlyProgressionMeterReadingsIsEnabled] = useState<boolean>()
+
+    const initialValues = {
+        onlyProgressionMeterReadingsIsEnabled: get(mobileConfig, 'onlyProgressionMeterReadingsIsEnabled'),
+    }
+
+    useEffect(() => {
+        if (mobileConfig) {
+            setOnlyProgressionMeterReadingsIsEnabled(mobileConfig.onlyProgressionMeterReadingsIsEnabled)
+        }
+    }, [])
+
+    const updateHook = MobileFeatureConfig.useUpdate({}, () => router.push('/settings?tab=mobileFeatureConfig'))
+    const updateAction = async (data) => {
+        await updateHook(data, mobileConfig)
+    }
+    const createAction = MobileFeatureConfig.useCreate({}, () => router.push('/settings?tab=mobileFeatureConfig'))
+    const action = mobileConfig ? updateAction : createAction
+
+    return useMemo(() => (<FormWithAction
+        initialValues={initialValues}
+        action={action}
+        colon={false}
+        layout='horizontal'
+        formValuesToMutationDataPreprocessor={(values) => {
+            if (!mobileConfig) {
+                values.organization = { connect: { id: userOrganizationId } }
+            }
+
+            values.onlyProgressionMeterReadingsIsEnabled = onlyProgressionMeterReadingsIsEnabled
```

Comment:
why don't you just use the value from the form?

You have a form with this field.  When you change it, you change the state above the form.  And here you put the value not from the form, but from the state. Why do that?

Reply:
you can use `valuePropName` for checkbox

Reply:
This is a bug fix. For some reason, the submit form had the previous value. Using the value from the state here solves the problem

Reply:
`valuePropName='checked'` doesn't help?

Reply:
unfortunately not working :(